skore/tests/unit/sklearn/estimator/feature_importance/test_coefficients.py

```diff
@@ -182,45 +182,97 @@ def test_estimator_report_coefficients_pandas_dataframe(estimator):
             id="PassiveAggressiveRegressor",
         ),
         pytest.param(sklearn.linear_model.Perceptron(), id="Perceptron"),
-        # pytest.param(sklearn.linear_model.PoissonRegressor(), id="PoissonRegressor"),
+        pytest.param(sklearn.linear_model.PoissonRegressor(), id="PoissonRegressor"),
         pytest.param(sklearn.linear_model.QuantileRegressor(), id="QuantileRegressor"),
         pytest.param(sklearn.linear_model.Ridge(), id="Ridge"),
         pytest.param(sklearn.linear_model.RidgeClassifier(), id="RidgeClassifier"),
         pytest.param(sklearn.linear_model.RidgeClassifierCV(), id="RidgeClassifierCV"),
         pytest.param(sklearn.linear_model.RidgeCV(), id="RidgeCV"),
         pytest.param(sklearn.linear_model.SGDClassifier(), id="SGDClassifier"),
-        # pytest.param(sklearn.linear_model.SGDOneClassSVM(), id="SGDOneClassSVM"),
+        pytest.param(sklearn.linear_model.SGDOneClassSVM(), id="SGDOneClassSVM"),
         pytest.param(sklearn.linear_model.SGDRegressor(), id="SGDRegressor"),
         pytest.param(sklearn.linear_model.TheilSenRegressor(), id="TheilSenRegressor"),
         pytest.param(sklearn.linear_model.TweedieRegressor(), id="TweedieRegressor"),
     ],
 )
 def test_all_sklearn_estimators(
-    request, estimator, regression_data, classification_data
+    request,
+    estimator,
+    regression_data,
+    positive_regression_data,
+    multi_regression_data,
+    classification_data,
+    outlier_data,
+    clustering_data,
 ):
     """Check that `coefficients` is supported for every sklearn estimator."""
+    multi = False
     if is_classifier(estimator):
         X, y = classification_data
     elif is_regressor(estimator):
-        X, y = regression_data
+        if isinstance(
+            estimator,
+            (
+                sklearn.linear_model.GammaRegressor,
+                sklearn.linear_model.PoissonRegressor,
+            ),
+        ):
+            X, y = positive_regression_data
+        elif isinstance(
+            estimator,
+            (
+                sklearn.linear_model.MultiTaskElasticNet,
+                sklearn.linear_model.MultiTaskElasticNetCV,
+                sklearn.linear_model.MultiTaskLasso,
+                sklearn.linear_model.MultiTaskLassoCV,
+                sklearn.cross_decomposition.CCA,
+                sklearn.cross_decomposition.PLSCanonical,
+            ),
+        ):
+            X, y = multi_regression_data
+            multi = True
+        else:
+            X, y = regression_data
+    elif is_outlier_detector(estimator):
+        X, y = outlier_data
+    elif is_clusterer(estimator):
+        X, y = clustering_data
```

Comment:
I'm thinking about the same here. We don't have that much stuff tested for outlier detector or clustering method. I'm thinking that we should probably add those tests at the moment that we make sure that the rest of the `EstimatorReport` is actually working.